src/main/groovy/com/bmuschko/gradle/docker/DockerSpringBootApplicationPlugin.groovy

```diff
@@ -1,34 +0,0 @@
-package com.bmuschko.gradle.docker
-
-import com.bmuschko.gradle.docker.internal.MainClassFinder
-import groovy.transform.CompileStatic
-import org.gradle.api.model.ObjectFactory
-import org.gradle.api.plugins.ExtensionAware
-
-/**
- * Opinionated Gradle plugin for creating and pushing a Docker image for a Spring Boot application.
- * <p>
- * This plugin can be configured with the help of {@link DockerSpringBootApplication}.
- *
- * @since 3.4.5
- */
-@CompileStatic
-class DockerSpringBootApplicationPlugin extends DockerConventionJvmApplicationPlugin<DockerSpringBootApplication> {
```

Comment:
I suggest don't delete original file. I know git sometimes do it because you change the content, in this case you should rename file first, commit it, and later change the content, it will save the history of the file.